fapolicy_analyzer/glade/main_window.glade

```diff
@@ -176,6 +176,39 @@
                     </child>
                   </object>
                 </child>
+                <child>
+                  <object class="GtkMenuItem">
+                    <property name="visible">True</property>
+                    <property name="can-focus">False</property>
+                    <property name="label" translatable="yes">System</property>
+                    <child type="submenu">
+                      <object class="GtkMenu">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <child>
+                          <object class="GtkImageMenuItem" id="fapdStopMenu">
+                            <property name="label">Stop Daemon</property>
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <property name="use-underline">True</property>
+                            <property name="use-stock">True</property>
+			    <signal name="activate" handler="on_fapdStopMenu_activate" swapped="no"/>
```

Comment:
Start and Stop are always active, regardless of the daemon status.  Since we know the status it would make sense to only enable the one that can be acted upon.

Reply:
Modified main_window to update Menu item sensitivity with fapolicyd icon updates. Currently testing the rpm installation on an FC34 VM.

Reply:
Rpm installation, running through Polkit, no issues observed wrt fapd control and monitoring.